src/Traits/JsonSerializable.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/*
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * This software consists of voluntary contributions made by many individuals
+ * and is licensed under the MIT license.
+ */
+
+namespace Somnambulist\Collection\Traits;
+
+/**
+ * Trait JsonSerializable
+ *
+ * @package    Somnambulist\Collection\Traits
+ * @subpackage Somnambulist\Collection\Traits\JsonSerializable
+ *
+ * @property array $items
+ */
+trait JsonSerializable
+{
+
+    /**
+     * @link http://php.net/manual/en/jsonserializable.jsonserialize.php
+     *
+     * @return string
+     */
+    public function jsonSerialize()
+    {
+        return \json_encode($this->items);
```

Comment:
This should call `toJson()` from the `Exportable` trait. Otherwise the output between the two methods will be different. `toJson()` exports to an array first before serializing the result.